src/Onboarding/Onboarding1.js

```diff
@@ -11,48 +14,59 @@ export default function Onboarding1(props) {
   const [partnerLastName, setPartnerLastName] = useState('');
 
   return (
-    <div>
-      <div>
-        <h1>Congratulations</h1>
-        <h2>
-          Tell us about yourself so we can
-          {'\n'}customize your planning experience.
-        </h2>
-        <h2>My name is</h2>
-        <input
-          type="text"
-          required
-          value={firstName}
-          onChange={(e) => setFirstName(e.target.value)}
-          placeholder="First Name"
-        />
-        <input
-          type="text"
-          required
-          value={lastName}
-          onChange={(e) => setLastName(e.target.value)}
-          placeholder="Last Name"
-        />
-        <h2>And my partner&apos;s name is</h2>
-        <input
-          type="text"
-          required
-          value={partnerFirstName}
-          onChange={(e) => setPartnerFirstName(e.target.value)}
-          placeholder="First Name"
-        />
-        <input
-          type="text"
-          required
-          value={partnerLastName}
-          onChange={(e) => setPartnerLastName(e.target.value)}
-          placeholder="Last Name"
-        />
-        <Link to="/onboarding-2">
-          <Button>Next</Button>
-        </Link>
-      </div>
-      <div>Right side image placeholder</div>
-    </div>
+    <Container fluid>
+      <Row style={{ height: '100vh', marginLeft: '15%' }}>
```

Comment:
Margin can be implemented using a Bootstrap utility class. If there are no other Rows, does this need to be set to 100vh, rather than taking the size of its content?